src/components/dashboardCards/ExperiencesTable.jsx

```diff
@@ -80,14 +80,7 @@ class ExperiencesTable extends Component {
                         </td>
                       </tr>
                     ))
-                  ) : (
-                    <tr>
-                      <td>Garrett Winters</td>
-                      <td>Video</td>
-                      <td>2013/07/01</td>
-                      <td>63</td>
-                    </tr>
-                  )}
+                  ) : null}
```

Comment:
What do we end up rendering where there is no media like this? Does it show up as null on the website, or is it just empty?

Reply:
It shows up as an empty table with just titles.